docs/API.md

```diff
@@ -107,17 +107,23 @@ A `<Link>` can know when the route it links to is active and automatically apply
 
 #### Props
 ##### `to`
-The path to link to, e.g. `/users/123`.
-
-##### `query`
+Can be either a string or an object.
```

Comment:
What about this:

> ##### `to`
> The value of `to` depends on what `history` library you are using. Unless you are using a custom `history` it will be treated as a [location descriptor](https://github.com/mjackson/history/blob/master/docs/Location.md#location-descriptors).